operator/controller/src/test/java/io/apicurio/registry/operator/it/AppFeaturesITTest.java

```diff
@@ -0,0 +1,63 @@
+package io.apicurio.registry.operator.it;
+
+import io.apicurio.registry.operator.EnvironmentVariables;
+import io.apicurio.registry.operator.api.v1.ApicurioRegistry3;
+import io.apicurio.registry.operator.api.v1.spec.AppFeaturesSpec;
+import io.apicurio.registry.operator.resource.ResourceFactory;
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static io.apicurio.registry.operator.api.v1.ContainerNames.REGISTRY_APP_CONTAINER_NAME;
+import static io.apicurio.registry.operator.resource.app.AppDeploymentResource.getContainerFromDeployment;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@QuarkusTest
+public class AppFeaturesITTest extends ITBase {
+
+    private static final Logger log = LoggerFactory.getLogger(AppFeaturesITTest.class);
+
+    @Test
+    void testAllowDeletesTrue() {
+        ApicurioRegistry3 registry = ResourceFactory
+                .deserialize("/k8s/examples/simple.apicurioregistry3.yaml", ApicurioRegistry3.class);
```

Comment:
This is a good place to use the example file.

Reply:
The example file was only intended for me to use in my local minikube environment, not for testing and not for inclusion in the repo.  My bad.  I meant to pull it out somewhere else and forgot.  Originally I had just duplicated the simple one as a starting point.